src/include/btree_inline.h

```diff
@@ -1525,14 +1525,24 @@ __wt_ref_block_free(WT_SESSION_IMPL *session, WT_REF *ref)
 
 /*
  * __wt_page_del_active --
- *     Return if a truncate operation is active.
+ *     Return if a truncate operation is active. "active" means approximately that the truncate is
+ *     still in progress, that is, that the underlying original page may still be required. This
+ *     function in practice is actually a visibility test (it returns whether the truncate is *not*
+ *     visible) and should be renamed and have its sense flipped to be more consistent with the rest
+ *     of the system.
```

Comment:
I want it to be its own ticket so it gets eyeballed carefully, because it's really easy to mess up making that kind of change. :-|
(also because it should create no functional change; having it be standalone makes that easier to crosscheck in the future)